pkgs/top-level/pidgin-packages.nix

```diff
@@ -0,0 +1,70 @@
+{ lib
+, newScope
+, pidgin
+, texlive
+, config
+}:
+
+lib.makeScope newScope (self:
+  let callPackage = self.callPackage;
+  in {
+    inherit callPackage;
+    pidgin = callPackage ../applications/networking/instant-messengers/pidgin {
+      withOpenssl = config.pidgin.openssl or true;
+      withGnutls = config.pidgin.gnutls or false;
+      plugins = [];
+      inherit (self) pidgin;
```

Comment:
probably not needed because callPackage calls with self

Reply:
#til